mission_data/category_links.yaml

```
@@ -166,4 +170,12 @@ category_links:
       - mission_slug: tiktok-like
-        sort_no: 200
\ No newline at end of file
+        sort_no: 200
+  - category_slug: support-on-threads
+    missions:
+      - mission_slug: follow-anno-threads
+        sort_no: 100
+      - mission_slug: threads-like
+        sort_no: 200
+      - mission_slug: threads-post
+        sort_no: 300
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)